charts/kangal/README.md

```diff
@@ -1,30 +1,38 @@
-# Kangal chart
-
-[Kangal](https://github.com/hellofresh/kangal) is a tool to spin up an isolated environment in a Kubernetes cluster to run performance tests using JMeter
-
-## TL;DR;
-```console
-$ helm repo add kangal https://hellofresh.github.io/kangal
-$ helm install kangal/kangal --name kangal
-$ kubectl apply -f  https://raw.githubusercontent.com/hellofresh/kangal/master/charts/kangal/crd.yaml
-```
+# Kangal Chart
+[Kangal](https://github.com/hellofresh/kangal) is a tool to spin up an isolated environment in a Kubernetes cluster to run performance tests using different load test providers.
 
 ## Introduction
-
 This chart bootstraps a [kangal](https://github.com/hellofresh/kangal) deployment using the [Helm](https://helm.sh) package manager.
 
 ## Prerequisites
-
-- Helm 2.13
+- Helm 3+
 - Kubernetes 1.12+
 
 ## Installing the Chart
-To install the chart with the release name `kangal` and specific [Kangal version](https://github.com/hellofresh/kangal/releases):
-
-```console
+To add the the repository to Helm:
+```shell
 $ helm repo add kangal https://hellofresh.github.io/kangal
-$ helm install --name kangal --set proxy.image.tag=1.0.0 --set controller.image.tag=1.0.0 kangal/kangal
-$ kubectl apply -f  https://raw.githubusercontent.com/hellofresh/kangal/master/charts/kangal/crd.yaml
+```
+
+To install the Custom Resource Definition:
+```shell
+$ kubectl apply -f https://raw.githubusercontent.com/hellofresh/kangal/master/charts/kangal/crd.yaml
+```
+
+To install the chart with the release name `kangal`:
+```shell
+$ helm install \
+  --set environment=dev \
```

Comment:
do we need to mark it as dev here?

Reply:
we need to set `environment`, as it is required by the chart and does not have default value.

I use `dev` as value to indicate that installation must be reviewed when in production, any suggestion?

Reply:
Got it, okay